dynomite/Cargo.toml

```diff
@@ -25,8 +25,8 @@ futures = "0.3"
 log = "0.4"
 # Disable default features since the `rustls` variant requires it. We re-enable `default` in our
 # `default` build configuration - see the [features] below.
-rusoto_core = { version = "0.48", optional = true, default_features = false }
-rusoto_dynamodb = { version = "0.48", optional = true, default_features = false }
+rusoto_core = { git = "https://github.com/sameer/rusoto.git", rev = "0346b41b414d036eb54cdf64781a17ba4b1a8ba4", optional = true, default_features = false }
+rusoto_dynamodb = { git = "https://github.com/sameer/rusoto.git", rev = "0346b41b414d036eb54cdf64781a17ba4b1a8ba4", version = "0.48", optional = true, default_features = false }
```

Comment:
Would it be possible to have this fork live under the `getditto` organization?

Reply:
Yes, I don't have perms to create public repos under the getditto org so I created it under my own. If you have perms, would you mind creating it? I can push the patch there